.github/workflows/prereleases.yml

```diff
@@ -40,6 +40,9 @@ jobs:
         env:
           NODE_ENV: "production"
           CI_OS: ${{ runner.os }}
+          # this is the "test/staging" key for sparrow analytics
+          # This is needed for create-cloudflare
+          SPARROW_SOURCE_KEY: "5adf183f94b3436ba78d67f506965998"
```

Comment:
Wrangler will re-build on publish later: https://github.com/cloudflare/workers-sdk/pull/6807/files#diff-d222a98ebbb50d0533a810ea1520cd202df58f31ee3edf09d0e9f407f3ccd08bR52-R60

Reply:
just for my understanding...why is one workflow using prod and the other staging?